CONTRIBUTING.md

```diff
@@ -24,8 +24,33 @@ For more on how 10up writes and manages code, check out our [10up Engineering Be
 
 ## Workflow
 
-The `develop` branch is the development branch which means it contains the next version to be released. `trunk` contains the latest released version.  Always work on the `develop` branch and open up PRs against `develop`.
+The `develop` branch is the development branch which means it contains the next version to be released. `trunk` contains the latest released version.
+
+### Feature branches
+
+1. From the `develop` branch create a new feature branch with the following naming structure: `feature|fix/[github-issue-number]--[branch-description]` (i.e. `feature/10--accordion-component-build`).
+2. Do work on the `feature` or `fix` branch. When finished, submit a pull request to merge your work back into the `develop` branch. In the PR, link to the issue (see the sidebar options on the right).
+3. If there are conflicts between your branch and the code that now exists on `develop` branch, merge `develop` into your branch to reintegrate new changes and resolve any conflicts.
+4. Assign a minimum on 1 reviewer to the pull request. At least one engineer must review all PRs.
+5. If any updated components (`/packages`) that will need to be relased to npm exist, be sure to update package version. numbers in their respective `package.json` files.
```

Comment:
I don't think this will work well. There might be multiple PRs making changes to the same package and merging something to `develop`  doesn't mean it goes straight to npm. So with this workflow we might end up bumping packages version multiple times before we even release to NPM. Typically we would want to group a few PRs before releasing a new package version.

My suggestion here is to do the process of updating package versions separately once we're ready to publish a new npm release. At the feature branch stage, we only have PR authors add the CHANGELOG entries to the PR template.

Example:
- 4 PRs are merged to develop, 2 packages updated.
- Once we're ready to publish to npm, a `maintainer/owner` creates a PR against `trunk` from `develop` or a `release` branch, manually update the package version to the next desired version and update the CHANGELOG.md.  Then publish to npm and merge into trunk.

Reply:
Documentation updated. Please review @nicholasio